Meta/import_wpt_test.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
```

Comment:
Filename: almost all files in `Meta/` use dashes, not underscores in their filename.

Reply:
Renamed 👍